src/main/java/com/minecolonies/coremod/colony/buildings/workerbuildings/BuildingSawmill.java

```diff
@@ -107,7 +107,7 @@ public boolean canRecipeBeAdded(final IToken token)
         {
             if(!ItemStackUtils.isEmpty(stack))
             {
-                if (stack.getItem().isIn(ItemTags.PLANKS) || stack.getItem().isIn(ItemTags.LOGS))
+                if (stack.getItem().isIn(ItemTags.PLANKS) || stack.getItem().isIn(ItemTags.LOGS) || stack.getItem().getRegistryName().getPath().contains("cactus"))
```

Comment:
It'd be better to add the cactus blocks to the item tags for planks and logs. 
or in the least make custom tags for them, because string comparisons are going to break easily